samples/csharp/MicrosoftEduGraphSamples/Workflows/ModuleWorkflow.cs

```diff
@@ -25,12 +25,13 @@ public ModuleWorkflow(IConfiguration configuration)
         /// <summary>
         /// Workflow to create and publish the module
         /// </summary>
-        public async Task ClassworkAsync()
+        /// <param name="appOnly">True value authenticates the graph client with application permissions only, otherwise it will be created with delegated permissions.</param> 
+        public async Task ClassworkAsync(bool appOnly = false)
         {
             try
             {
                 // Get a Graph client using delegated permissions
```

Comment:
// Get a Graph client based on the appOnly parameter

Reply:
updated the comment to "// Get a Graph client based on the appOnly parameter" on line 33